app/controllers/items_controller.rb

```diff
@@ -24,4 +24,8 @@ def create
   def item_params
     params.require(:item).permit(:name, :description, :price, :email, :image)
   end
+
+  def available_items
+    Item.all.where(available: true)
```

Comment:
Also, please make this a scope on Item so you can to `Item.available` http://guides.rubyonrails.org/active_record_querying.html#scopes